env.ts

```diff
@@ -43,11 +43,10 @@ const MAILCHIMP_HOST =
   process.env.NEXT_PUBLIC_MAILCHIMP_HOST! ??
   'https://norge.us6.list-manage.com/subscribe/post?u=215524fdb6&amp;id=ba4ff3ce11';
 const DATAJEGER_EMAIL_ADDRESS =
-  process.env.NEXT_PUBLIC_DATAJEGER_EMAIL_ADDRESS! ??
-  'datajegeren.staging@norge.no';
+  process.env.NEXT_PUBLIC_DATAJEGER_EMAIL_ADDRESS! ?? 'datajegeren@digdir.no';
 const FDK_MAIL_SERVICE_ENDPOINT =
   process.env.FDK_MAIL_SERVICE_ENDPOINT! ??
-  'mail-sender-service.staging.fellesdatakatalog.digdir.no/api/sendmail';
+  'mail-sender-service.fellesdatakatalog.digdir.no/api/sendmail';
```

Comment:
Vi hadde visst aldri fått ingress på denne.
```suggestion
  'http://35.244.241.109/api/sendmail';
```

Reply:
Bruker staging likevel. Løst på annen måte